src/utils/Cookie.php

```diff
@@ -37,6 +37,10 @@ public static function get(InternetAddress $address) : int{
         return 0;
     }
 
+    public static function hasServerSecurity () : bool {
+        return self::$serverHasSecurity;
```

Comment:
changing a static field to a static getter doesn't change the fact that it is still a static field...

Reply:
> changing a static field to a static getter doesn't change the fact that it is still a static field...

I need to make changes on [PocketMine-MP](https://github.com/pmmp/PocketMine-MP) at the same time, how do I do that?
I'm thinking of adding it to [YML Server Properties](https://github.com/pmmp/PocketMine-MP/blob/stable/src/YmlServerProperties.php).

Reply:
I don't see what PM has to do with anything. It should be a switch in the RakLib Server constructor.

Reply:
> I don't see what PM has to do with anything. It should be a switch in the RakLib Server constructor.

there should be a place where the user can set this

Reply:
This is a library, the user of the library is who implement it, a switch in the RakLib server constructor give to user the ability to change it.